server/index.js

```diff
@@ -9,13 +9,21 @@ const PORT = 3002;
 // TODO: update once static client files are available
 // app.use(express.static())
 app.use(express.json());
-app.use(express.urlencoded());
+app.use(express.urlencoded({ extended: true }));
 
 app.get('/api/menu/:id', (req, res) => {
   const { id } = req.params;
-  res.send(id);
+
+  db.find({ restaurant_id: id }, (err, data) => {
+    if (err) {
+      res.send(404);
```

Comment:
Are 404s the only status code that you will need to send back if there's an error?

Reply:
I believe thats appropriate here, but could be wrong. 404 refers to a resource not being found. For our purposes, that should suffice.